conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,192 @@
+
+## (WIP) Part III: How do I write quick start content?
+This section is a work in progress. Keep an eye out for a more detailed style guide!
+
+### How to write a task
+In order to write a task well, try to answer the following questions first:
+1. What is the goal of the task? Why should the user do this?
+2. What are the required steps to take in order to complete the task?
+3. Are there any optional steps?
+4. Can the user validate they’ve completed the task? How?
+
+### Calling out UI elements
+When writing out UI elements in the quick start instructions, follow these guidelines:
+
+* **Buttons, dropdowns, tabs, fields, and other UI controls:** As it’s written in the UI + bolded.
+* **Everything else, including page, window, and panel names:** As it’s written in the UI. If sentence case, use double quotes.
+* **Code or user-entered text:** Monospaced font
+* **CLI commands:** Monospaced font. “In running text, use a bold monospaced font for a command. If a parameter or option is a variable value, use an italic monospaced font; otherwise, use a bold monospaced font for the parameter and a monospaced font for the option.”
+
+## (WIP) Part IV: How can I contribute a quick start?
+This section is a work in progress. Keep an eye out for more detailed contribution processes to come!
+
+### Internal contributions
+
+#### Process goals:
+* To provide user-centric, functional quick starts for OpenShift users across a wide range of product areas
+* To create a successful collaboration across teams (UXD, CCS, PM, etc.) for every quick start
+* To define which teams are involved at each step of the quick start effort so that the end result is organized, clean, and in final form for users
+* To respect the bandwidth and input of each team
+
+#### Phase 1: Planning
+1. Determine which product area needs a quick start.
+  * Identify stumbling blocks or complex areas where we can enable user success with as little friction as possible. This information will be gathered from:
+    * UXD user research
+    * Pain points shared from internal feedback sessions (Solution Architects, Technical Marketing, CEE/Customer Support, etc.)
+    * Product Management input
+    * OpenShift or Kubernetes documentation
+    * Features we want to highlight or push more to entice users to try
+    * Areas we want to offer more opinionated views or flows. For example, there are likely cases where there are multiple ways for users to accomplish a task, but we’re recommending a method or suggesting how to get up and running with XYZ.
+  * **Team responsible:** Product Management; UXD Design; possibly UXD Research
+  * **Time frame:** Before epic prioritization
+  * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start.
+2. Plan user flows and interaction design.
```

Comment:
Creating mockups for this is time consuming.  Since these will be CRs it will may be quick to create the content and get a quick preview.  May be something to look into!

Reply:
Makes total sense - I can see the guess and check working just fine possibly!